dockers/docker-sonic-gnmi/gnmi-native.sh

```diff
@@ -124,7 +124,11 @@ else
 fi
 
 USER_AUTH=$(extract_field "$GNMI" '.user_auth')
-if [ ! -z "$USER_AUTH" ] && [  $USER_AUTH != "null" ]; then
+# If user_auth is not set, default to certs
+if [ $USER_AUTH == "null" ]; then
+    USER_AUTH="cert"
+fi
+if [ ! -z "$USER_AUTH" ] && [  $USER_AUTH != "null" ] && [  $USER_AUTH != "none" ]; then
```

Comment:
Not necessary to check this: [ $USER_AUTH != "null" ] because above code already make sure USER_AUTH always not "null"